tests/ui_tests/app/page/common/configuration/page.py

```diff
@@ -167,7 +167,17 @@ def fill_password_and_confirm_fields(self, password: str, confirmation: str, dis
     @allure.step('Click on group {title}')
     def click_on_group(self, title: str):
         """Click on group with given title"""
-        self.find_and_click(self.locators.group_btn(title))
+
+        def check_group_expanded(group: WebElement):
+            return "expanded" in group.get_attribute("class")
```

Comment:
You name it "check", but no check in body, so better to name it `is_group_expanded` or even `is_expanded_in_group_class`

Reply:
changed